conferences/types.go

```diff
@@ -47,3 +47,165 @@ type ConferenceSlot struct {
 	// issue sponsor tickets and those cannot be bought individually)
 	AvailableToPublic bool
 }
+
+// ClaimPayment represents a payment for N claims
+type ClaimPayment struct {
+	ID uint64 `gaum:"field_name:id"`
```

Comment:
As we are not using gaum anymore, could we remove these struct tags?

```suggestion
	ID uint64
```